iamy/aws.go

```diff
@@ -334,6 +336,17 @@ func (a *AwsFetcher) populateIamData(resp *iam.GetAccountAuthorizationDetailsOut
 		a.data.addRole(&role)
 	}
 
+	policyArns := make([]*string, 0)
+	for _, policyResp := range resp.Policies {
+		policyArns = append(policyArns, policyResp.Arn)
+	}
```

Comment:
If we appended the string, instead of the pointer to the string, here we wouldn't need to convert this to a list of strings later on?